src/commands.rs

```diff
@@ -8,7 +8,7 @@ mod sgx;
 mod softsign;
 mod start;
 mod version;
-#[cfg(feature = "yubihsm")]
+#[cfg(feature = "yubihsm-client")]
```

Comment:
why this renaming?

Reply:
Because there is a  crate called `yubihsm` in `[dependencies]` section and cargo treats it as a feature, the feature name and crate name should not be the same. When I want to add an extra crate such as `signatory` into the feature, there are two chooses: 
- One,  make a new feature such as `yubihsm-client`; 
- two, rename the crate `yubihsm` in dependencies and add  a feature called `yubihsm` in `[features]` section. 

If I use the second method, I can rename back to `yubihsm` in the `lib.rs` file by using `extren crate yubihsm-renamed as yubihsm`, but bad luckly, there is a mod  called `yubihsm` in the project which name is a conflict with the crate, so I can't do like that. Then I have to change the `crate::yubihsm` into `crate::yubihsm_renamed` to make it work which will change much a lot than the first method. 

So I chose the first method.